src/test/java/BuyTshirtProcessTest.java

```diff
@@ -37,28 +33,20 @@ public static void setupWebDriver() {
   @BeforeMethod
   public void setup() {
 
-    catalog = new SaucedomoCatalogPage();
-    checkOut = new SaucedomoCheckOutPage();
+
     logIn = new SaucedomoLogInPage();
-    shoppingCart = new SaucedomoShoppingCartPage();
-    verification = new SaucedomoVerificationPage();
-    completed = new SaucedomoCompletedOrderPage();
   }
 
   @Test(description = "This test is in charge of navigating through the webpage and"
       + "simulating the process of acquisition for a t-shirt")
   public void buyTshirtTest() {
-    logIn.visitLogin();
-    logIn.logIn("standard_user", "secret_sauce");
-
-    catalog.selectItem();
-    catalog.goToShoppingCart();
-
-    shoppingCart.goToCheckOut();
-
-    checkOut.fillOutForm("Juan", "Palma", "77601");
+    completed = logIn.visitLogin()
+        .logIn("standard_user", "secret_sauce")
+        .selectItem()
+        .goToShoppingCart()
+        .goToCheckOut()
+        .fillOutForm("Juan", "Palma", "77601").finishProcess();
 
-    verification.finishProcess();
 
     Object container = completed.getCompletionContainer();
     String result = completed.getContainerHeader();
```

Comment:
integra porfa esto tambien en el fluent y que al final solo tengas la variable result

Reply:
moved to the next commit